doc/source/release-notes/Percona-XtraDB-Cluster-8.0.26-16.1.rst

```diff
@@ -0,0 +1,48 @@
+.. _PXC-8.0.26-16:
+
+================================================================================
+*Percona XtraDB Cluster* 8.0.26-16
+================================================================================
+
+:Date: 2022
+:Installation: `Installing Percona XtraDB Cluster <https://www.percona.com/doc/percona-xtradb-cluster/8.0/install/index.html>`_
+
+Percona XtraDB Cluster (PXC) supports critical business applications in your public, private, or hybrid cloud environment. Our free, open source, enterprise-grade solution includes the high availability and security features your business requires to meet your customer expectations and business goals.
+
+Release Highlights
+================================================================================
+
+The following are some of the notable fixes for MySQL 8.0.26, provided by Oracle, and included in this release:
+
+* Deprecation of the `TLSv1 and TLSv1.1 <https://tools.ietf.org/id/draft-ietf-tls-oldversions-deprecate-02.html>`__ connection protocols.
+* Identifiers with certain terms are updated. See `Functionality Added or Changed section in the 8.0.26 Release Notes <https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-26.html#mysqld-8-0-26-feature>`__ for a list of updated identifiers. The following terms have been changed:
+    - “master” is changed to “source”
+    - “slave” is changed to “replica”
+    - “mts”, for “multithreaded slave”, is changed to “mta” for “multithreaded applier”.
+
+For more information, see the `MySQL 8.0.26 Release Notes <https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-26.html>`__
+
+
+Bugs Fixed
+================================================================================
+
+* :jirabug:`PXC-3824`: Incorrect directive in Systemd Unit File (Thanks to Jim Lohiser for reporting this issue)
+* :jirabug:`PXC-3706`: Fix for a race condition in group commit queue (Thanks to Kevin Sauter for reporting this issue)
+* :jirabug:`PXC-3628`: Server allows mysql.wsrep_* tables to be converted to MyISAM
+* :jirabug:`PXC-3731`: A fix for deleting data from the source but not from the replica, the ``sql_log_bin=0`` command causes the data to replicate inside the cluster, but also writes the data to the binary log.
+* :jirabug:`PXC-3857`: The following system variables are renamed:
+    - ``wsrep_slave_threads`` is changed to ``wsrep_applier_threads``
+    - ``wsrep_slave_FK_checks`` is changed to ``wsrep_applier_FK_checks``
+    - ``wsrep_slave_UK_checks`` is changed to ``wsrep_applier_UK_checks``
+    - ``wsrep_restart_slave`` is changed to ``wsrep_restart_replica``
+
+
+
+Known Issues (unfixed problems that you should be aware of)
+================================================================================
+
+* :jirabug:`PXC-3039`: No useful error messages if an SSL-disabled node tries to join an SSL-enabled cluster
+* :jirabug:`PXC-3092`: Log a warning at startup if a keyring is specified, but cluster traffic encryption is turned off
```

Comment:
https://jira.percona.com/browse/PXC-3092 was fixed in 8.0.23. So it can be removed from the "Known Issues" list.

Reply:
Done.